legend-engine-xt-persistence-cloud-pure/src/main/resources/core_persistence_cloud/external/format/awspersistence/integration.pure

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import meta::external::persistence::aws::cnas::generation::*;
```

Comment:
We don't have the guidelines for code layout clearly written down as yet .. so we can pick this up in the next refactoring. 

What do you think about a new Maven module per cloud type ? E.g legend-engine-xt-persistence-cloud-pure-aws, legend-engine-xt-persistence-cloud-pure-gcp ?

Reply:
Yeah - this is something David and I have talked about before. Think it makes sense to go there when we do the next one? Definitely need a better layout here.